platform/security/policy/security-policy-context/src/test/java/org/codice/security/policy/context/PolicyManagerTest.java

```diff
@@ -127,6 +153,33 @@ public void setup() {
     }
 
     @Test
+    public void testSimpleAttributeMappings() {
+
+        for (Map.Entry<String, List<ContextAttributeMapping>> entry : simpleAttributeMap.entrySet()) {
+            ContextPolicy policy = manager.getContextPolicy(entry.getKey());
+            CollectionPermission permission = policy.getAllowedAttributePermissions();
+
+            Assert.assertTrue(permission.implies(entry.getValue()
```

Comment:
We really should move these assertions to `assertThat`
https://codice.atlassian.net/wiki/display/DDF/Unit+Testing